sky/clouds/kubernetes.py

```diff
@@ -241,6 +240,10 @@ def make_deploy_resources_variables(
             # TODO(romilb): Allow user to specify custom images
             'image_id': image_id,
         }
+
+        if resources.ports is not None:
+            deploy_vars['ports'] = resources.ports
+
```

Comment:
Is this redundant?

Reply:
Done in https://github.com/skypilot-org/skypilot/pull/2713/commits/d3fc952568a4ddf853355b15006b42fecd737e32